File: app/src/main/java/com/bopr/android/smailer/ui/RecipientsActivity.java
package com.bopr.android.smailer.ui;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

/**
 * Recipients list activity.
 *
 * @author Boris Pronin (<a href="mailto:boprsoft.dev@gmail.com">boprsoft.dev@gmail.com</a>)
 */
public class RecipientsActivity extends AppActivity {

    @NonNull
    @Override
    protected Fragment createFragment() {
        return new RecipientsFragment();
    }

}

File: app/src/main/kotlin/com/bopr/android/smailer/ui/RecipientsActivity.kt
package com.bopr.android.smailer.ui

import androidx.fragment.app.Fragment

/**
 * Recipients list activity.
 *
 * @author Boris Pronin ([boprsoft.dev@gmail.com](mailto:boprsoft.dev@gmail.com))
 */
class RecipientsActivity : BaseAppActivity() {

    override fun createFragment(): Fragment {
        return RecipientsFragment()
    }
}